### PR TITLE
Bug/SK-507 | Add functionality to inspect combiner queue state (debug mode)

### PR DESCRIPTION
**Pull request by ahellander**
## Description

Also implements feature SK-503.

### CODE REVIEW

File: fedn/fedn/network/combiner/round.py [hunk 1]
```diff
@@ -331,13 +331,11 @@ def execute_training_round(self, config):
             model_id = str(uuid.uuid4())
             self.modelservice.set_model(a, model_id)
             a.close()
+            data['model_id'] = model_id
```
Comment (Wrede) [inline fedn/fedn/network/combiner/round.py:334]: report server status if model is None? So one can debug...

**Review by Wrede — APPROVED** [review]
approved but with comment.

**Comment by Wrede** [comment]
This is not a bug. Think you have mixed up issues IDs.

File: fedn/fedn/network/combiner/round.py (hunk 1 above)
Comment (ahellander) [reply fedn/fedn/network/combiner/round.py:334]: This is reported on line 325.

**Pull request merged**